app/validators/routes_and_redirects_validator.rb

```diff
@@ -21,7 +27,7 @@ def validate(record)
       redirects_must_not_have_routes(record, routes)
       redirects_must_include_base_path(record, redirects)
     else
-      routes_must_include_base_path(record, routes)
+      routes_must_include_base_path(record, routes) unless is_unpublishing
```

Comment:
I added a `base_path` method on `Unpublishing`s to support the `base_path` check. Personally I like the idea of a single validation class that supports both `Editions` and `Unpublishing`, perhaps I can add methods to both `Edition` and `Unpublishing` which supports the differences between the two classes while allowing the validator to be the same (therefore allowing us to validate against any new classes by only implementing those methods). What do you think?